lib/column/enum.go

```diff
@@ -47,6 +48,9 @@ func Enum(chType Type, name string) (Interface, error) {
 			enum.iv[values[i]] = proto.Enum8(v)
 			enum.vi[proto.Enum8(v)] = values[i]
 		}
+		enum.minEnum = int8(slices.Min(maps.Keys(enum.vi)))
+		enum.maxEnum = int8(slices.Max(maps.Keys(enum.vi)))
+		enum.continuous = (enum.maxEnum-enum.minEnum)+1 == int8(len(enum.vi))
```

Comment:
For Enum8 non-continuous & continous could both use a 256 bitmap (only 32 bytes, could skip heap by being in struct as `[4]uint64`)

Reply:
Not sure I understand your suggestion. Do you have some (semi-code) example of how you would use that, and which library? (Since I don't know of any bitmap implementations in the Golang standard library itself.)

Reply:
No need to use library, can be done in loop above, something like
```
var bitset [4]uint64
for i := range values {
  bitIndex := uint8(indexes[i])
  bitset[bitIndex>>3] |= 1 << (bitIndex&7)
}
```